src/current/_includes/v23.1/ui/transactions-filter.md

```diff
@@ -6,14 +6,39 @@
   {% assign page_prefix = "ui-" %}
 {% endif %}
 
+## Transaction Fingerprints results
+
+The Transaction Fingerprints returned are determined by the selected **Search Criteria**:
+
+1. By default, the **Top** `100` Transaction Fingerprints **By** `% of All Runtime` are returned.
+  - Change the number returned by selecting `25`, `50`, or `500` from the dropdown.
+  - Change the column filter by selecting `CPU Time`, `Contention Time`, `Execution Count`, or `Transaction Time` from the dropdown.
+1. Select the [**Time Range**](#time-interval).
+1. Click **Apply**.
+
+    The list of transactions that satisfy the search criteria is displayed. The results can be further [searched and filtered](#search-and-filter).
+
 ### Time interval
 
 To view transaction fingerprints within a specific time interval, click the time interval selector and pick an interval. The time interval field supports preset time intervals (1 Hour, 6 Hours, 1 Day, etc.) and custom time intervals. To select a custom time interval, click the time interval field and select **Custom time interval**. In the **Start (UTC)** and **End (UTC)** fields select or type a date and time.
 
-Use the arrow keys to cycle through previous and next time intervals. When you select a time interval, the same interval is selected in the [Metrics]({{ link_prefix }}ui-overview.html#metrics) page.
+Use the arrow keys to cycle through previous and next time intervals. Click **Now** to select the most recent interval. When you select a time interval, the same interval is selected in the [Metrics]({{ link_prefix }}ui-overview.html#metrics) page.
```

Comment:
Same feedback as statement fingerprints section re arrow buttons and sentence structure